tests/subsys/net/lib/downloader/src/main.c

```diff
@@ -960,6 +960,8 @@ static ssize_t z_impl_zsock_recvfrom_http_redirect_and_close(
 		return 0;
 	}
 
+	memcpy(buf, HTTP_HDR_REDIRECT, strlen(HTTP_HDR_REDIRECT));
+	return strlen(HTTP_HDR_REDIRECT);
 	return 0;
```

Comment:
This is unreachable and should be removed.

Reply:
Removed.